CRM/Contribute/BAO/Contribution.php

```diff
@@ -4486,7 +4485,6 @@ public static function completeOrder($input, &$ids, $objects, $transaction = NUL
     $changeDate = CRM_Utils_Array::value('trxn_date', $input, date('YmdHis'));
 
     $contributionResult = self::repeatTransaction($contribution, $input, $contributionParams);
-    $contributionParams['financial_type_id'] = $contribution->financial_type_id;
```

Comment:
@mattwire with the other merged I'm just removing the setting & unsetting of $contributionParams['financial_type_id']